src/app/legacy/components/ScrollablePromo/__snapshots__/index.test.jsx.snap

```diff
@@ -94,12 +94,12 @@ exports[`ScrollablePromo it should match a11y snapshot for list 1`] = `
 }
 
 .emotion-2 {
-  display: block;
```

Comment:
Is this a change on the orginal scrollable component? If so, we should also a11y swarm this.